Userland/Libraries/LibWeb/CSS/StyleProperties.h

```diff
@@ -192,7 +183,10 @@ class StyleProperties : public RefCounted<StyleProperties> {
     // FIXME: This needs protection from GC!
     JS::GCPtr<CSS::CSSStyleDeclaration const> m_animation_name_source;
 
-    PropertyValues m_property_values;
+    Array<RefPtr<CSS::StyleValue const>, number_of_properties> m_property_values;
+    Array<u8, ceil_div(number_of_properties, 8uz)> m_property_important {};
+    Array<u8, ceil_div(number_of_properties, 8uz)> m_property_inherited {};
```

Comment:
Any reason for not using an `AK::Bitmap` for each of these? I guess the downside to that is it doesn't have inline storage. :thinking:

Reply:
Right, bitmap would be two more mallocs per instance

Reply:
A zuteilung-frei version might be comfy, my feeling is that a lot of uses of Bitmaps are places where we know statically how large they need to be.